packages/xo-server-test/src/_xoConnection.js

```diff
@@ -130,6 +131,15 @@ class XoConnection extends Xo {
     return remote
   }
 
+  async createTempServer(params) {
+    const id = await this.call('server.add', params).catch(err =>
+      console.warn('server.add', err)
+    )
+    this._tempResourceDisposers.push('server.remove', { id })
+    await fromEvent(this._objects, 'finish')
```

Comment:
We'll wait indefinitely (until the jest timeout) if the server got a connection troubles.